src/content/datachannel/datatransfer/index.html

```diff
@@ -60,11 +60,11 @@
         <div class="label">Send progress: </div>
         <progress id="sendProgress" max="0" value="0"></progress>
       </div>
-
       <div class="progress">
         <div class="label">Receive progress: </div>
         <progress id="receiveProgress" max="0" value="0"></progress>
       </div>
+      <div id="bitrate"></div>
```

Comment:
To get consistent layout, probably best to parallel the progress divs, something like this:

    <div>
        <div class="label">Current bitrate: </div>
        <div id="bitrate"></div>
     </div>